core/app/identity.go

```diff
@@ -0,0 +1,3 @@
+package app
+
+var RuntimeIdentity = []byte("runtime")
```

Comment:
What is this needed? Runtime name is in runtime directly (https://github.com/cosmos/cosmos-sdk/blob/main/runtime/types.go#L11)

Reply:
Actually never mind, I see where it is used and it would be a cyclic import.

Reply:
it will be needed for consensus messages as well. Modules should check to make sure the signer is runtime